hack/examples/run-gradle.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
yeah an e2e with this 

of course you would use the sample pipeline the maven based example I initially did uses